apps/api/src/app/workflows-v2/usecases/validate-content/prepare-and-validate-content/prepare-and-validate-content.usecase.ts

```diff
@@ -292,12 +297,48 @@ export class PrepareAndValidateContentUsecase {
         controlValueToContentIssues,
         item,
         StepContentIssueEnum.MISSING_VALUE,
-        `${toSentenceCase(item)} is missing`,
+        `${toSentenceCase(item)} is required`,
         item
       );
     }
   }
 
+  private overloadMinValueIssues(
+    jsonSchema: JSONSchemaDto,
+    controlValues: Record<string, unknown>,
+    contronValueIssues: Record<string, ContentIssue[]>
+  ) {
+    if (typeof jsonSchema !== 'object') {
+      return;
+    }
+
+    let schemaToExtractDefaults: JSONSchemaDefinition = jsonSchema;
+    if (jsonSchema.anyOf && jsonSchema.anyOf.length > 0) {
+      schemaToExtractDefaults =
+        jsonSchema.anyOf.find((item) => {
+          return isMatchingJsonSchema(item, controlValues);
+        }) ?? jsonSchema[0];
+    }
+    const minValuesFromSchema = extractMinValuesFromSchema(schemaToExtractDefaults);
```

Comment:
Find the schema that matches with control values and extract the min values for all fields.